tests/emitters/emitter_base_test.py

```diff
@@ -101,3 +101,20 @@ def test_array_bound_bad_entry_fails(archive_fixture):
     bounds[0] = (-1, 0, 1)  # Invalid entry.
     with pytest.raises(ValueError):
         GaussianEmitter(archive, x0, 1, bounds=bounds)
+
+
+### x0 should be 1-dimensional ###
```

Comment:
Move this info to the test name -- the test name should (ideally) have enough info that you can tell what was failing just by reading the name, e.g. `test_emitters_fail_when_x0_not_1d`